releasenotes/notes/cluster-agent-collect-pod-disruption-budget-5d8cf316c3633684.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    The cluster-agent now pod disruption budgets from the cluster.
```

Comment:
```suggestion
    The cluster-agent now applies a pod disruption budget policy from the cluster.
```

Reply:
Thanks for pointing out the omission. I went with

```
    The cluster-agent now can collect pod disruption budgets from the cluster.
```